docs/samples/client/csharp/SampleService/main.tsp

```diff
@@ -36,31 +36,27 @@ enum Versions {
 }
 
 @doc("float fixed enum")
-@fixed
```

Comment:
this decorator has been deprecated for a long time.
Somehow the pipeline reports it cannot build therefore I just removed them